tests/client/amo/test_utils.js

```diff
@@ -1,15 +1,50 @@
 import createStore from 'amo/store';
+import * as featuredActions from 'amo/actions/featured';
 import * as landingActions from 'amo/actions/landing';
 import * as api from 'core/api';
 import {
+  ADDON_TYPE_EXTENSION,
   ADDON_TYPE_THEME,
   SEARCH_SORT_POPULAR,
   SEARCH_SORT_TOP_RATED,
 } from 'core/constants';
-import { loadLandingAddons } from 'amo/utils';
+import { loadFeaturedAddons, loadLandingAddons } from 'amo/utils';
 
 
 describe('amo/utils', () => {
+  describe('loadFeaturedAddons()', () => {
+    const addonType = ADDON_TYPE_EXTENSION;
+    let ownProps;
+
+    beforeEach(() => {
+      ownProps = {
+        params: {
+          application: 'android',
+          visibleAddonType: 'extensions',
+        },
+      };
+    });
+
+    it('requests a large page of featured add-ons', () => {
+      const store = createStore({ application: 'android' });
+      store.dispatch(featuredActions.getFeatured({ addonType }));
+      const mockApi = sinon.mock(api);
+      const entities = sinon.stub();
+      const result = { results: [] };
+
+      mockApi
+        .expects('featured')
+        .once()
+        .withArgs({ api: {}, filters: { addonType, page_size: 25 } })
+        .returns(Promise.resolve({ entities, result }));
+
+      return loadFeaturedAddons({ store, params: ownProps.params })
```

Comment:
This looks like it should be a test for `fetchFeaturedAddons` instead since it sets up mock expectations for the API call. I don't see `fetchFeaturedAddons` tested directly though and I commented above that you could just merge it into `loadFeaturedAddons` anyway and be done (unless I'm missing something).